src/common.c

```diff
@@ -186,15 +186,19 @@ psf_log_printf (SF_PRIVATE *psf, const char *format, ...)
 					d = va_arg (ap, int) ;
 
 					if (d < 0)
-					{	d = -d ;
-						sign_char = '-' ;
+					{	sign_char = '-' ;
 						if (lead_char != '0' && left_align == SF_FALSE)
 							width_specifier -- ;
-						} ;
+						
```

Comment:
(Nit-pick) (Not showing up in GitLab, even with "Hide Whitespace Changes" disabled.) Trailing whitespace added on line 192.